src/main/resources/db/migration/V201908061557__Constraints.sql

```diff
@@ -1,8 +1,10 @@
 alter table schema_data add constraint schema_data_pk primary key (id);
 
+delete from users where SCHEMA_ID not in (select id from SCHEMA_DATA);
```

Comment:
Har du forsøkt å kjøre denne som en select for å se om du får riktig resultat?
altså `select * from users....`